proton.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env @PYTHON_NAME@
+#!/usr/bin/env python
```

Comment:
is this still necessary given the proton sh script?

Reply:
I think it's good practice to leave it here, and it allows directly executing the script if you feel so inclined (or if distributions end up packaging `Proton`, they can just skip the shell script and rename this). But yes, it isn't strictly necessary.